### ISSUE TITLE
when someone ge in the room what i am writing in my chatbox disappearing 

### ISSUE THREAD

**Issue by DoublebassOO** [elieserdejesus/JamTaba#1082]
When writing a message in the chatbox and someone get in the room  all is deleted , and i need to write it again ..;

jamtaba 2.4 as vst in reaper

**Comment by elieserdejesus** [comment]
this issue was reported in https://github.com/elieserdejesus/JamTaba/issues/947, but I tested and not see the bug. I check again...

**Comment by elieserdejesus** [comment]
**Investigating the issue**:

- Tested using 2 computers.
- No problems in Standalone, it's a VST issue only.
- When the 2nd computer connect in the server the chat input text is loosing the focus
- In my test the chat text is not wiped, but maybe the **loose focus** issue is causing unexpected behavior.

**Comment by DoublebassOO** [comment]
yes you re right Elieser,  not wiped just  not able to see it . when i send it even if not visible  the text is send correctly. but hard to remember what we were typing . But scrolling in the chatbox make the text come back again.  and yes it was in the vst version.
thks
